pages/resources/testnet.md

```diff
@@ -10,8 +10,8 @@ import EVMAssets from '../../components/evm/assets'
 
 | Variable              | Value     |
 | --------------------- | --------- |
-| `axelar-core` version | `v0.31.2` |
-| `vald` version        | `v0.31.2` |
+| `axelar-core` version | `v0.32.0` |
+| `vald` version        | `v0.32.0` |
```

Comment:
this should only be changed tomorrow, since it tracks the current version of the network